src/components/DatePicker/DatePicker.tsx

```diff
@@ -66,7 +72,7 @@ export default function DatePicker(props: Props): JSX.Element {
           value={temporalValue}
           onChange={(newValue: any) => {
             setTemporalValue(newValue);
-            props.onChange(newValue?.isValid() ? newValue?.toDate() : null);
+            props.onChange(newValue?.isValid() ? newValue?.format('YYYY-MM-DD') : null);
```

Comment:
this looks like a breaking change, if client previously handled a JS object, it won't handle this new change. 
thoughts on reverting this change and have client call util to get the formatted value?

Reply:
When returning the Date object, the timeZone was lost in the client, that's why I returned the string here. If will try using fromJS in the client and see what happens

Reply:
Ok I think I may have missed the objective here. We could possibly return the timeZone as a param in the onChange.
So existing clients that depend on getting back a JS Date in UTC will not be affected.
basically, onChange(jsdate, timeZone)
Thoughts?

Reply:
We already have the timeZone in the client, actually the client is passing the timeZone to the datepicker. So I don't think we need to do that. The problem is that when we set the value with getTodaysDateFormatted() we are setting a string, and when the value comes from the onChange it's a date. So I wanted to return a string here to make that cleaner, but I could try creating a new getTodaysDate function that returns a date, and then use .fromJSDate() with the timeZone in the client. What do you think?